pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -106,6 +106,9 @@ func (c *cephStatusChecker) checkStatus() {
 	status, err = cephclient.StatusWithUser(c.context, c.clusterInfo)
 	if err != nil {
 		logger.Errorf("failed to get ceph status. %v", err)
+		if err := c.updateCephStatus(cephStatusOnError(err.Error())); err != nil {
```

Comment:
What is the `err` when the status request fails? We likely need to wrap it with something more descriptive.

Reply:
`message: 'failed to get status. . timed out: exit status 1'`

Reply:
Isn't that enough?

Reply:
Ah, the `StatusWithUser()` already wrapped it with a decent message. What's up with the extra `.`? It doesn't look very good.

Reply:
Yes, I know, it's just that the buffer is empty... I can fix that.

Reply:
Fixed